runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -90,6 +112,26 @@ protected void shutDown() throws Exception
 		overlayManager.remove(overlay);
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
```

Comment:
If you toggle the config while wearing the item, it won't get readded until next inventory change